### PR TITLE
Prepare Documentation for MeiliSearch v0.13

### PR DESCRIPTION
**Pull request by bidoubiwa**
This needs to be added to the documentation 
- [x] Explain how to update MeiliSearch in installation and in faq /meilisearch/MeiliSearch#794
- [x] Change the settings behavior meilisearch/meilisearch#824
    - Remove acceptNewFields routes
    - Change information in `/references/settings`, `/references/settings/searchable-attributes`, `/references/settings/displayed-attributes`, `/guides/advanced_guides/field_properties`
    - ~fieldsDistribution in the GET indexes/:uid/stats route should now returns all the known attributes no matter the displayedAttributes are.~ **not in this version**
- [x] Remove the sys-info routes meilisearch/meilisearch#810
    - Needs to be removed from the api references here: https://docs.meilisearch.com/references/sys-info.html
    - And from the sidebar in `.vuepress/config.js`
- [x] Placeholder search meilisearch/meilisearch#771 
   - Changes in the [search api references](https://docs.meilisearch.com/references/search.html) #417 
   - Changes in the [Search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) #417
- [x] Post Search Route meilisearch/meilisearch#735
    - Changes in the [search api references](https://docs.meilisearch.com/references/search.html) #416
   - Changes in the [Search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) #416
   - Changes in the code-sample file: `.code-samples.meilisearch.yaml` #416
- [x] chore(http): do not require auth on /health endpoint meilisearch/meilisearch#823
   - In the [authentication guide](https://docs.meilisearch.com/guides/advanced_guides/authentication.html#master-key) you should add `/health` to the `public` key and also create a `no key` bullet where you can also add `/health`
- [x] feat(sentry): make sentry dsn customizable meilisearch/meilisearch#813
   - Add cli and env variables to the [configuration](https://docs.meilisearch.com/guides/advanced_guides/configuration.html#passing-arguments-via-the-command-line)

### CODE REVIEW

**Comment by MarinPostma** [comment]
> fieldsDistribution in the GET indexes/:uid/stats route should now returns all the known attributes no matter the displayedAttributes are.

nope, not yet, this will be in another version, it's too much work for now.

File: guides/advanced_guides/search_parameters.md [hunk 1]
```diff
@@ -18,11 +18,15 @@ Search parameters let the user customize their search request.
 
 ## Query (q)
 
-The query parameter is the **only mandatory** parameter. This is the string used by the search engine to find relevant documents.
+This is the string used by the search engine to find relevant documents.
+
+When q isn't specified, a placeholder search is performed, returning documents that best match other search parameters.
+A placeholder allows you to find documents without providing search terms. It will return documents sorted according to the [ranking rules](/guides/advanced_guides/settings.md#ranking-rules) order.
+It also support [faceting](/guides/advanced_guides/faceted_search.md) and [filtering](/guides/advanced_guides/filtering.md)
```
Comment (curquiza) [inline guides/advanced_guides/search_parameters.md:25]:
```suggestion
It also supports [faceting](/guides/advanced_guides/faceted_search.md) and [filtering](/guides/advanced_guides/filtering.md)
```

File: guides/advanced_guides/search_parameters.md [hunk 2]
```diff
@@ -18,11 +18,15 @@ Search parameters let the user customize their search request.
 
 ## Query (q)
 
-The query parameter is the **only mandatory** parameter. This is the string used by the search engine to find relevant documents.
+This is the string used by the search engine to find relevant documents.
+
+When q isn't specified, a placeholder search is performed, returning documents that best match other search parameters.
```
Comment (curquiza) [inline guides/advanced_guides/search_parameters.md:23]:
```suggestion
When `q` isn't specified, a placeholder search is performed, returning documents that best match other search parameters.
```

**Comment by bidoubiwa** [comment]
> A question: how both of the search methods (GET and POST) are going to be handle in code-sample? Are the current search examples going to be in POST example? 

Yes, I will make an issue to find out how to best showcases this. But it will be for a later pr

**Pull request merged**